db/migrate/20170626122608_set_public_body_translation_defaults.rb

```diff
@@ -0,0 +1,20 @@
+# -*- encoding : utf-8 -*-
+class SetPublicBodyTranslationDefaults < ActiveRecord::Migration
+  def up
+    change_column :public_body_translations,
+                  :short_name, :text, :null => false, :default => ""
+    change_column :public_body_translations,
+                  :notes, :text, :null => false, :default => ""
+    change_column :public_body_translations,
+                  :publication_scheme, :text, :null => false, :default => ""
+  end
```

Comment:
Annoyingly we have a load of data that would make this migration fail:

```sql
SELECT COUNT(*) FROM public_body_translations WHERE name IS NULL;
 count
-------
   268

SELECT COUNT(*) FROM public_body_translations WHERE short_name IS NULL;
 count
-------
   308
```

I think we need a rake task to clean up existing records, _then_ apply these migrations.